src/main/java/Storage.java

```diff
@@ -0,0 +1,115 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * Represents a Storage.
+ * Able to retrieve and write data to a storage txt file stored locally.
+ */
+public class Storage {
+    private String path;
+    private File file;
+    private boolean isExisted;
+
+    Storage(String path) {
+        File f = new File(path);
+        if (f.exists()) {
+            this.file = f;
+            this.path = path;
+            this.isExisted = true;
+        } else {
+            try {
+                f.getParentFile().mkdirs();
+                f.createNewFile();
+                this.file = f;
+                this.path = path;
+                this.isExisted = false;
+            } catch (IOException e) {
+                System.out.println("An error occured");
+                e.printStackTrace();
+            }
+        }
+    }
+
+    /**
+     * Return boolean if the storage txt file already exists.
+     *
+     * @return Boolean value of if the storage txt file already exists.
+     */
+    public boolean getExisted() {
```

Comment:
Perhaps this method name can be renamed to something like `hasExisting()` to more clearly indicate that it returns a boolean?